Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM ubuntu:hirsute-20210723
+FROM ubuntu:jammy-20221130
```

Comment:
Is there a reason we don't use the LTS?

Reply:
I updated the `Dockerfile` locally as follows and the image works just fine:

```Dockerfile
FROM ubuntu:20.04

# Disable DL3008 as it is not possible to pin virtual packages such as qemu-kvm
# hadolint ignore=DL3008
RUN apt-get update \
 && DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends \
   ansible \
   git \
   qemu-kvm \
   qemu-utils \
   unzip \
   xorriso \
   curl \
   jq \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
```

Question: Why pin the version for specific packages?